tests/files/odl/feedbooks_bibliographic.atom

```diff
@@ -254,46 +254,54 @@
+<id>http://www.feedbooks.com/item/3264308</id>
+<dcterms:identifier xsi:type="dcterms:URI">urn:ISBN:9781338331042</dcterms:identifier>
+<author>
+  <name>Angie Sage</name>
+  <uri>https://www.feedbooks.com/search?query=contributor%3A%22Angie+Sage%22</uri>
+</author>
+<published>2020-01-23T15:43:59Z</published>
+<updated>2020-01-23T15:48:46Z</updated>
+<dcterms:language>en</dcterms:language>
+<dcterms:publisher>Scholastic Inc. Audio</dcterms:publisher>
+<dcterms:issued>2019-07-31</dcterms:issued>
+<summary>Mega bestselling author Angie Sage takes flight with an epic adventure that imagines dragons in the modern world.
+
+Once our world was full of dragons who lived in harmony with humans. Dragons counseled kings and queens. They fought in human battle...</summary>
+<dcterms:extent>517 MB</dcterms:extent>
+<category scheme="http://www.feedbooks.com/categories" term="FBFIC000000" label="Fiction"/>
+<category scheme="http://www.feedbooks.com/categories" term="FBJUV000000" label="Juvenile &amp; Young Adult"/>
+<category scheme="http://www.feedbooks.com/categories" term="FBJUV037000" label="Fantasy, Magic"/>
+<schema:duration>1080</schema:duration>
+<schema:abridged>false</schema:abridged>
+<link type="text/html" title="View on Feedbooks" rel="alternate" href="https://www.feedbooks.com/item/3264308"/>
+<link type="image/jpeg" rel="http://opds-spec.org/image" href="http://covers.feedbooks.net/item/3264308.jpg?size=large&amp;t=1579794526"/>
+<link type="image/jpeg" rel="http://opds-spec.org/image/thumbnail" href="http://covers.feedbooks.net/item/3264308.jpg?size=large&amp;t=1579794526"/>
+<content type="html">&lt;p&gt;Mega bestselling author Angie Sage takes flight with an epic adventure that imagines dragons in the modern world.&lt;/p&gt;
+
+&lt;p&gt;Once our world was full of dragons who lived in harmony with humans. Dragons counseled kings and queens. They fought in human battles. And they gave humans the most precious gift of all: magic.&lt;/p&gt;
+
+&lt;p&gt;But after a group of rogue dragons, the Raptors, tried to take over Earth, all dragons were banished to another realm.&lt;/p&gt;
+
+&lt;p&gt;Most humans forgot about the dragons, claiming they never existed. Eleven-year-old Sirin knows the truth -- she grew up with stories passed down through the generations. However, when her mother falls ill, even Sirin has trouble believing in magic . . . until she sees a mysterious streak of silver in the night sky.&lt;/p&gt;
+
+&lt;p&gt;Sirin becomes the first child to "lock" with a dragon in centuries -- forming a deep friendship unlike anything she's ever imagined. But Sirin learns that not all dragons returned with good intentions, and soon she finds herself at the center of a battle between the dragons who want to protect the humans . . . and those who want to destroy them.&lt;/p&gt;</content>
+<odl:license>
+  <dcterms:identifier xsi:type="dcterms:URI">urn:uuid:551c7b6c-3e45-11ea-a2da-a26cdbaff453</dcterms:identifier>
```

Comment:
I think the real UUIDs should not be in github. I don't remember exactly why so you might want to check with Hadrien if it matters for this. Or maybe this is fake, but you made a more realistic fake one than I did :)

Reply:
It's better to replace it with a random UUID indeed.